src/minicraft/screen/TitleDisplay.java

```diff
@@ -242,13 +224,12 @@ public void render(Screen screen) {
 		"Milk is an idea!",
 		"Creeper, aw man",
 		"So we back in the mine,",
-		"pickaxe swinging from side to side",
+		"Pickaxe swinging from side to side",
 		"In search of Gems!",
 		"Life itself suspended by a thread",
 		"saying ay-oh, that creeper's KO'd!",
 		"Gimmie a bucket!",
 		"Farming with water!",
-		"Press \"R\"!",

```

Comment:
Hey this actually does something you know. Have you ever tried actually pressing r? I'd keep that one in.

Reply:
Not anymore ;)